stories/index.js

```diff
@@ -41,7 +41,25 @@ storiesOf('Picker', module)
       showPreview={boolean('Show preview', true)}
       showSkinTones={boolean('Show skin tones', true)}
       custom={CUSTOM_EMOJIS}
-    />
+    />))
+    .add('not-found-url', () => (
+      <Picker
+        onClick={action('clicked')}
+        onSelect={action('selected')}
+        onSkinChange={action('skin changed')}
+        native={boolean('Unicode', true)}
+        set={select('Emoji pack', SETS, SETS[0])}
+        emojiSize={number('Emoji size', 24)}
+        perLine={number('Per line', 9)}
+        title={text('Idle text', 'Your Title Here')}
+        emoji={text('Idle emoji', 'department_store')}
+        defaultSkin={number('Default skin tone', 1)}
+        color={color('Highlight color', '#ae65c5')}
+        showPreview={boolean('Show preview', true)}
+        showSkinTones={boolean('Show skin tones', true)}
+        custom={CUSTOM_EMOJIS}
+        notFound={{notFoundImg:'https://assets-cdn.github.com/images/icons/emoji/shipit.png?v7'}}
+      />
```

Comment:
I wonder if we need to add a whole new story for this feature, or if adding a knob is enough. 

Like you could do: 

```js
storiesOf('Picker', module)
  .addDecorator(withKnobs)
  .add('default', () => (
    <Picker
      onClick={action('clicked')}
      onSelect={action('selected')}
      onSkinChange={action('skin changed')}
      native={boolean('Unicode', true)}
      set={select('Emoji pack', SETS, SETS[0])}
      emojiSize={number('Emoji size', 24)}
      perLine={number('Per line', 9)}
      title={text('Idle text', 'Your Title Here')}
      emoji={text('Idle emoji', 'department_store')}
      defaultSkin={number('Default skin tone', 1)}
      color={color('Highlight color', '#ae65c5')}
      showPreview={boolean('Show preview', true)}
      showSkinTones={boolean('Show skin tones', true)}
      custom={CUSTOM_EMOJIS}
      notFound={{
        notFoundImg: text('Not found Image URL', '')
      }}
    />));
```

and you'd get a text field to paste any URL to see the effect of using this property.
![storybook-knob](https://user-images.githubusercontent.com/8042228/41994530-b8946f22-7a1c-11e8-87e2-fa6952324e82.gif)